composer.json

```diff
@@ -11,7 +11,8 @@
         "php": "^7.1.3",
```

Comment:
Here we are saying PHP 7.1, in Travis we're testing 7.2 and 7.3. Do we need to up the version here, or add 7.1 to Travis?